htdocs/compta/facture/card.php

```diff
@@ -2846,9 +2846,9 @@
 			{
 				print '<td class="nowrap" style="padding-left: 5px">';
 				$arraylist = array('amount' => $langs->transnoentitiesnoconv('FixAmount'), 'variable' => $langs->transnoentitiesnoconv('VarAmountOneLine', $langs->transnoentitiesnoconv('Deposit')));
-				print $form->selectarray('typedeposit', $arraylist, GETPOST('typedeposit'), 0, 0, 0, '', 1);
+				print $form->selectarray('typedeposit', $arraylist, $conf->global->PROPALE_DEPOSIT_TYPE, 0, 0, 0, '', 1);
```

Comment:
May be we should have here
GETPOSTISSET('typedeposit')?GETPOST('typedeposit'):$conf->global->PROPALE_DEPOSIT_TYPE
instead of 
$conf->global->PROPALE_DEPOSIT_TYPE

So the default value is set at first form creation, but choice is not lost if we submit form and there is an error

Reply:
Also, to have a common setup, we can name the default deposit variable into
MAIN_DEFAULT_DEPOSIT_TYPE
MAIN_DEFAULT_DEPOSIT_AMOUNT